src/components/votes.tsx

```diff
@@ -24,7 +24,7 @@ export class Votes extends React.Component<IVotesProps> {
 
       
     return (
-            <div className="flex-row">
+            <div className="flex-row" style={{display: "flex", flexDirection: "row", flexWrap:"wrap"}}>
```

Comment:
I would prefer custom styling to be in with its own class name in a scss/css file